### ISSUE TITLE
one-var rule doesn’t have default options

### ISSUE THREAD

**Issue by lo1tuma** [eslint/eslint#3449]
# Problem description

In my `.eslintrc` I’ve configured the `one-var` rule as `"one-var": 2`, according to the [rule documentation](http://eslint.org/docs/rules/one-var#options) the default option should be `"one-var": [2, "always"]`.

If I try to lint the following code with `"one-var":  2`

```js
var foo = 'a';
var bar = 'b';
```

I get no error. When I set the configuration to `"one-var": [2, "always"]` in my `.eslintrc` then I get the expected errors.

-----

## Reproducible in the online demo

No error:
```js
/*eslint "one-var": 2 */
var foo = 'a';
var bar = 'b';
```

Works as expected:
```js
/*eslint "one-var": [2, "always"] */
var foo = 'a';
var bar = 'b';
```

-----

Seems like the default options were removed in commit e4076d2855dcc1450d51c1bbe79b7749d546f052.

**Comment by gyandeeps** [comment]
What is wrong here code or docs?

**Comment by lo1tuma** [comment]
I would say the code, because the rule does nothing although if you explicitly enable the rule.

**Comment by gyandeeps** [comment]
With that being said, `always` should be applied to `var` , `let` and `const` when the config is like this
`/*eslint "one-var": 2 */`

**Comment by lo1tuma** [comment]
@gyandeeps yes, `"one-var": 2` should have the same behavior as `"one-var": [2 "always"]`.
